class LostAuthException(Exception):
    pass

class ReceiptFailed(Exception):
    pass
